resources/views/categoryShow.blade.php

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    @if (session('success'))
+        <div class="alert alert-success">
+            {{ session('success') }}
+        </div>
+    @endif
+    <div class="alert alert-success">
+        <p>Name: {{ $category->name }}</p>
+        <p>Description: {{ $category->description }}</p>
+        {{-- @if ($category->icon)
+            <img src="{{ asset('storage/' . $category->icon) }}" alt="Category Icon" width="100">
+        @endif --}}
+    </div>
+</body>
+</html>
```

Comment:
You don't need blade files for categories, you can use Postman to test the endpoints